bosshud/ze_insensible_sr8gm6yj12hs7_RG_v9_6.txt

```diff
@@ -0,0 +1,42 @@
+"math_counter"
+{
+	"config"
+	{
+		"BossBeatenShowTopDamage"					"0"

```

Comment:
What's the reasoning for this?

Reply:
The bosses (Boss1, Boss2 etc.) are probably appearing in the same bossfight in phases judging from observations in VIDE so to prevent confusion during the bossfight I disabled the top-damage
gametext